File: 프로그래머스/0/181836. 그림 확대/그림 확대.cs
using System;
using System.Collections.Generic;

public class Solution 
{
    public string[] solution(string[] picture, int k) 
    {
        List<string> pictureK = new List<string>();
        
        for(int i=0; i<picture.Length; i++)
        {
            string temp = "";
            string pictrueStr = picture[i];
            for(int j=0; j<picture[i].Length; j++)
            {
                for(int w=0; w<k; w++)
                {
                    temp += pictrueStr[j].ToString();
                }
            }
            
            for(int g=0; g<k; g++)
            {
                pictureK.Add(temp);
            }            
        }
        return pictureK.ToArray();
    }
}

File: 프로그래머스/0/181836. 그림 확대/그림 확대.cpp
#include <string>
#include <vector>

using namespace std;

vector<string> solution(vector<string> picture, int k) 
{
    vector<string> vec;
    for(int i=0; i<picture.size(); i++)
    {
        string temp = picture[i];
        string s = "";
        for(int j=0; j<temp.length(); j++)
        {
            for(int w=0; w<k; w++) { s += temp[j]; }
        }
        for(int g=0; g<k; g++) { vec.push_back(s); }
    }
    return vec;
}